pkg/systemd/quadlet/quadlet.go

```diff
@@ -969,7 +969,7 @@ func addNetworks(quadletUnitFile *parser.UnitFile, groupName string, serviceUnit
 }
 
 func getAbsolutePath(quadletUnitFile *parser.UnitFile, filePath string) (string, error) {
-	if !filepath.IsAbs(filePath) {
+	if filePath[0] != '%' && !filepath.IsAbs(filePath) {
```

Comment:
Shouldn't there be a special case for `%%` ?
```
$ man systemd.unit | grep "Single percent"
       │"%%"      │ Single percent sign                           │ Use "%%" in place of "%" to specify a single      │
```

Reply:
Addressed in the `startsWithSystemdSpecifier` function